src/style/core.ts

```diff
@@ -50,7 +50,11 @@ const root = css`
     ${lightTheme}
     --rdg-selection-color: #66afe9;
     --rdg-font-size: 14px;
-    --rdg-cell-frozen-box-shadow: calc(2px * var(--rdg-sign)) 0 5px -2px rgba(136, 136, 136, 0.3);
+    --rdg-cell-frozen-box-shadow: 2px 0 5px -2px rgba(136, 136, 136, 0.3);
+
+    :dir(rtl) {
```

Comment:
```suggestion
    &:dir(rtl) {
```

Reply:
It is supported in all the browsers 
> Baseline 2023
Newly available
Since December 2023, this feature works across the latest devices and browser versions. This feature might not work in older devices or browsers.

This is the only place where `--rdg-sign` variable is used